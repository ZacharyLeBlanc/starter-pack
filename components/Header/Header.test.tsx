import React from 'react';
import { render } from '@testing-library/react';

import { Header } from './index';

describe('Snapshot Tests', () => {
  it('loads and displays greeting', () => {
    const { container } = render(<Header />);
    expect(container).toMatchSnapshot();
  });
});
