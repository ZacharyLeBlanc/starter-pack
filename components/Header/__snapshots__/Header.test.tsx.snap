// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Tests loads and displays greeting 1`] = `
<div>
  <h1
    class="text-5xl font-bold text-purple-500"
  >
    Hello world!
  </h1>
</div>
`;
